src/components/Tool/FixCommitmentInfo/FixCommitmentInfo.test.tsx

```diff
@@ -0,0 +1,191 @@
+    </ThemeProvider>
+  </SnackbarProvider>
+);
+
+describe('FixCommitmentContact', () => {
+  beforeEach(() => {
+    setContactFocus.mockClear();
+  });
+  it('default with test data', async () => {
+    const { getByText } = render(<Components />);
+    await waitFor(() => {
+      expect(getByText('Fix Commitment Info')).toBeInTheDocument();
+      expect(
+        getByText('You have 2 partner statuses to confirm.'),
+      ).toBeInTheDocument();
+    });
+  });
+
+  it('has correct styles', async () => {
+    const { getByTestId } = render(<Components />);
+    const home = getByTestId('Home');
+
+    expect(home).toHaveStyle('display: flex');
+
+    await waitFor(() => {
+      const container = getByTestId('Container');
+      const divider = getByTestId('Divider');
+      const description = getByTestId('Description');
+
+      expect(container.className).toEqual(expect.stringContaining('container'));
+      expect(container).toHaveStyle('width: 70%');
+
+      expect(divider.className).toEqual(expect.stringContaining('divider'));
+      expect(divider).toHaveStyle('margin-top: 16px');
+
+      expect(description.className).toEqual(
+        expect.stringContaining('descriptionBox'),
+      );
+
+      expect(description).toHaveStyle('margin-bottom: 16px');
+    });
+  });
+
+  it('Shows hide modal', async () => {
+    const { getAllByTestId, queryByText, getByText } = render(<Components />);
+
+    await waitFor(() => {
+      userEvent.click(getAllByTestId('hideButton')[0]);
+    });
+
+    expect(
+      getByText(
+        'Are you sure you wish to hide Tester 1? Hiding a contact in MPDX actually sets the contact status to "Never Ask".',
+      ),
+    ).toBeInTheDocument();
+
+    userEvent.click(getAllByTestId('action-button')[0]);
+
+    expect(
+      queryByText(
+        'Are you sure you wish to hide Tester 1? Hiding a contact in MPDX actually sets the contact status to "Never Ask".',
+      ),
+    ).not.toBeInTheDocument();
+  });
+
+  it('updates commitment info', async () => {
+    const { getAllByTestId, queryByText } = render(<Components />);
+
+    await waitFor(() => {
+      userEvent.click(getAllByTestId('confirmButton')[0]);
+    });
+
+    await waitFor(() =>
+      expect(
+        queryByText(
+          'Are you sure you wish to update Tester 1 commitment info?',
+        ),
+      ).toBeInTheDocument(),
+    );
+
+    userEvent.click(getAllByTestId('action-button')[1]);
+
+    await waitFor(() =>
+      expect(queryByText('Tester 1')).not.toBeInTheDocument(),
+    );
+
+    await waitFor(() => {
+      userEvent.click(getAllByTestId('confirmButton')[0]);
+    });
+
+    await waitFor(() =>
+      expect(
+        queryByText(
+          'Are you sure you wish to update Tester 2 commitment info?',
+        ),
+      ).toBeInTheDocument(),
+    );
+
+    await waitFor(() => {
+      userEvent.click(getAllByTestId('hideButton')[0]);
+    });
+
+    await waitFor(() =>
+      expect(
+        queryByText(
```

Comment:
`queryBy` is for when elements don't exist in the document.

```suggestion
        getByText(
```